src/com/xilinx/rapidwright/design/DesignTools.java

```diff
@@ -3299,13 +3295,23 @@ public static void createCeSrRstPinsToVCC(Design design) {
                     } else {
                         pinName = "RSTREGBU";
                     }
-                    if (si.getSitePinInst(pinName) == null) {
-                        vcc.createPin(pinName, si);
-                    }
+                    maybeCreateVccPinAndPossibleInversion(si, pinName, vcc, gndInvertibleToVcc);
                 }
             }
         }
     }
+    
+    private static void maybeCreateVccPinAndPossibleInversion(SiteInst si, String sitePinName, Net vcc, Net gndInvertibleToVcc) {
+        if (si.getSitePinInst(sitePinName) == null) {
+            SitePinInst sitePin = vcc.createPin(sitePinName, si);
+            if (gndInvertibleToVcc != null && sitePinName.contains("RST")) {
+                assert sitePin.getSiteWireBELPins().size() == 1;
+                BELPin pin = sitePin.getSiteWireBELPins().get(0);
+                assert pin.getBEL().getPins().length == 2;
+                si.routeIntraSiteNet(gndInvertibleToVcc, pin, pin);
```

Comment:
Looks like this only does intra-site routing for when GND is inverted to VCC. Do you want to do intra-site routing for when VCC is actually desired (e.g. for `CE`)?

Reply:
Oh wait, my mistake. `CKEN`-s do not have an inverter.